types/underscore/underscore-tests.ts

```diff
@@ -1,3 +1,324 @@
+// common testing types and objects
+interface SimpleStringObject {
+    a: string;
+    b: string;
+}
+
+const simpleStringObjectArray: SimpleStringObject[] = [{ a: 'a', b: 'c' }, { a: 'b', b: 'b' }, { a: 'c', b: 'a' }];
```

Comment:
I had originally provided real values here because I thought it might be beneficial to also write JS tests to confirm that these types were accurate at runtime, but per a conversation with @jgonggrijp that's not worth doing. Would it be best to switch these to `declare` statements instead of providing real JS initializations?

Reply:
Understood, thanks!